tokio/CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# 0.2.11 (January 27, 2019)
+
+### Fixes
+- docs: misc fixes and tweaks (#2155, #2103, #2027, #2167, #2175).
+- macros: handle generics in `#[tokio::main]` method (#2177).
+- sync: `broadcast` potential lost notifications (#2135).
+- rt: improve "no runtime" panic messages (#2145).
+
+### Added
+- optional support for using `parking_lot` internally (#2164).
```

Comment:
should this note that there is a feature flag to enable this?

Reply:
I try to keep it short w/ a link to find out more. It's just a change log.